crates/host/src/wasmbus/mod.rs

```diff
@@ -1882,6 +1887,43 @@ impl Host {
             }
         });
 
+        let config_watch: JoinHandle<anyhow::Result<_>> = spawn({
```

Comment:
So I actually got rid of this because the config bundles do the caching/watching for us now. I figured the ctl interface could do a passthrough to the KV bucket. Is the concern here around how often wadm might be querying the config? I actually think we are ok there too.

If we do want to go back to this, we should probably change out how the config bundle works. Otherwise we end up with possibly _two_ whole copies of all config in memory

Reply:
@thomastaylor312 gotcha, then I think we may not have the config bundle fully implemented since I wasn't getting config updates before I re-added this. Let me take a second look and see if I just missed something simple with the config generator